### PR TITLE
chore(testing): restore overview tests

### PR DESCRIPTION
**Pull request by miguel-crespo-fdc**
Ref: SRX-9PBRYS

### CODE REVIEW

File: pkg/db/db.go [hunk 1]
```diff
@@ -2439,12 +2439,6 @@ func (h *DBHandler) DBWriteDeployment(ctx context.Context, tx *sql.Tx, deploymen
 	if err != nil {
 		return fmt.Errorf("could not write deployment into DB. Error: %w\n", err)
 	}
-	if !skipOverview {
```
Comment (miguel-crespo-fdc) [inline pkg/db/db.go:2442]: Does nothing

File: pkg/db/db_test.go [hunk 2]
```diff
@@ -2596,39 +2596,12 @@ func TestReadWriteOverviewCache(t *testing.T) {
 										Argocd:           &api.EnvironmentConfig_ArgoCD{},
 										EnvironmentGroup: &dev,
 									},
-									//Applications: map[string]*api.Environment_Application{
```
Comment (miguel-crespo-fdc) [inline pkg/db/db_test.go:2599]: Read and write overview cache, we can simply ignore the information that was deleted

File: pkg/db/overview_test.go [hunk 3]
```diff
@@ -362,260 +362,6 @@ func TestUpdateOverviewEnvironmentLock(t *testing.T) {
 	}
 }
 
-func TestUpdateOverviewDeployment(t *testing.T) {
```
Comment (miguel-crespo-fdc) [inline pkg/db/overview_test.go:365]: Moved to overview_test.go

File: services/cd-service/pkg/service/overview_test.go [hunk 4]
```diff
@@ -1266,343 +1244,460 @@ func TestOverviewServiceFromCommit(t *testing.T) {
...
+				if err := repo.Apply(testutil.MakeTestContext(), tr); err != nil {
+					t.Fatal(err)
+				}
+			}
+
+			svc := &OverviewServiceServer{
+				Repository: repo,
+				Shutdown:   shutdown,
+				DBHandler:  repo.State().DBHandler,
+				Context:    context.Background(),
```
Comment (sven-urbanski-freiheit-com) [inline services/cd-service/pkg/service/overview_test.go:1391]: we should use the same context as above (ctx)

**Review by sven-urbanski-freiheit-com — APPROVED** [review]

**Pull request merged**